soc/nxp/rw/power.c

```diff
@@ -27,11 +28,36 @@ LOG_MODULE_DECLARE(soc, CONFIG_SOC_LOG_LEVEL);
 
 power_sleep_config_t slp_cfg;
 
+#if DT_NODE_HAS_STATUS(DT_NODELABEL(pin1), okay)
```

Comment:
why is there only code for pin1?

Reply:
Thanks, I have changed this to include pin 0

Reply:
I meant in general in the PR not just this line

Reply:
I have tested only for Pin 1 with the EVK. We can add Pin 0 once it can be tested.

Reply:
okay